indico/modules/events/contributions/util.py

```diff
@@ -16,6 +16,7 @@
 
 from __future__ import unicode_literals
 
+import re
```

Comment:
As you can see in the failing tests on Travis, we are using isort to ensure consistent sorting of imports. Please fix the order.